src/t8_forest/t8_forest_iterate.cxx

```diff
@@ -420,15 +428,134 @@ t8_forest_iterate_replace (t8_forest_t forest_new,
     T8_ASSERT (eclass == t8_forest_get_tree_class (forest_old, itree));
     ts = t8_forest_get_eclass_scheme (forest_new, eclass);
     T8_ASSERT (ts == t8_forest_get_eclass_scheme (forest_new, eclass));
+
+    /* Refine flag of -3 is not defined. Its set here for testing reasons.
+     * later in */
+    refine = -3;
```

Comment:
Please avoid magic-numbers. Do we use it in a test? 
Please see my above comment regarding refinement/coarsen/delete-values. This parameter should be added. If this parameter is not needed anymore, remove it.